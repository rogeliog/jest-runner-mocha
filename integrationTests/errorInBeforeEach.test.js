const runJest = require('./runJest');

it('Works when it has an error inside of beforeEach', () => {
  return expect(runJest('errorInBeforeEach')).resolves.toMatchSnapshot();
});
