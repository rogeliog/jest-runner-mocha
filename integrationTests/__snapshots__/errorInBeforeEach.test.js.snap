// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works when it has an error inside of beforeEach 1`] = `
"FAIL integrationTests/__fixtures__/errorInBeforeEach/__mocha__/__tests__/file.test.js
    My tests Nested describe \\"before each\\" hook for \\"This nested test passes\\"
Error: Error in nested beforeEach
      at mocked-stack-trace
  My tests
    ✓ This test passes
  Nested describe
    My tests
      ✕ \\"before each\\" hook for \\"This nested test passes\\"
Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 passed, 2 total
Snapshots:   0 total
Time:
Ran all test suites.

"
`;
